messages/help.txt

```diff
@@ -22,10 +22,10 @@ Dúfame, že si v tom každý nájde svoje :slightly_smiling_face:
 
 *HAMKA*
 `http://hamka.sk/`
-Objednávať si môžte do 9:30 v tvare ham+"číslo", polievka je súčasťou
+Objednávať si môžte do 9:40 v tvare ham+"číslo", polievka je súčasťou
```

Comment:
9:30 was the correct time, see https://github.com/vacuumlabs/obedbot/blob/master/src/server.js#L46

Reply:
OK, staff handbook is wrong. I've created PR.